src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -8389,6 +8393,7 @@ class ContainerAppsJob(TrackedResource):  # pylint: disable=too-many-instance-at
         "extended_location": {"key": "extendedLocation", "type": "ExtendedLocation"},
         "identity": {"key": "identity", "type": "ManagedServiceIdentity"},
         "provisioning_state": {"key": "properties.provisioningState", "type": "str"},
+        "running_status": {"key": "properties.runningStatus", "type": "str"},
```

Comment:
Please add YAML test to cover it. And rerun all tests relate to `az containerapp job create/update --yaml`

Reply:
It is a readonly property, we cannot update it with `az containerapp job create/update --yaml`, right?

Reply:
Why you need to update this class?
In fact `az containerapp job create/update --yaml` is using `class Job`, not this one. I think you don't need to update this file.